ngraph_bridge/ngraph_encapsulate_impl.cc

```diff
@@ -135,8 +134,6 @@ Status NGraphEncapsulateImpl::GetNgExecutable(
     try {
       ng_exec = backend->compile(ng_function);
     } catch (const std::exception& ex) {
-      string fn_name = ng_function->get_friendly_name();
-      NgraphSerialize("tf_function_" + fn_name + ".json", ng_function);
```

Comment:
plot_graph in case of failure?

Reply:
May be add fn_name to the error message, that way we'll know which file to look at.

Reply:
> plot_graph in case of failure?

I think it's of limited use to dump the graph on error with there being limited debug info in the graphs at the moment. For now, dumping graphs is explicit through `NGRAPH_TF_DUMP_GRAPHS`.

> May be add fn_name to the error message, that way we'll know which file to look at.

Good suggestion. I've added function name to the error message.